docs/src/main/sphinx/functions/array.rst

```diff
@@ -60,6 +60,10 @@ Array functions
 
     Returns an array of elements in ``x`` but not in ``y``, without duplicates.
 
+.. function:: array_histogram(x) -> map(K, bigint)
```

Comment:
This return type of `map(K, bigint)` is not rendering for me in safari or chrome. The same issue happens for the existing `histogram` function from the [Aggregate functions page](https://trino.io/docs/current/functions/aggregate.html#map-aggregate-functions) (shown below in a screenshot). I will look into this rendering issue separate from this PR.
<img width="577" alt="image" src="https://user-images.githubusercontent.com/10050520/217455268-8cf09a6d-5656-494a-bb76-9685e21b00a6.png">

Reply:
This is likely something me and @mosabua need to look into. Please create another issue around this and tag us in it please!